README.md

```diff
@@ -71,6 +71,34 @@ Before you can run the PerfKit Benchmaker on Cloud providers you need accounts a
 You also need the software dependencies, which are mostly command line tools and credentials to access your
 accounts without a password.  The following steps should help you get the CLI tool auth in place.
 
+If you are running on Windows, you will need to install Github Windows
+since it includes tools like openssl and an ssh client. Alternatively you can
+install Cygwin since it should include the same tools.
+
+## Install python 2 (and pip)
```

Comment:
Nit: captialize Python. Recommend specifying Python 2.7 as well - we use features not available in earlier versions.